docs/ops-recipe/harborDownOrUnhealthy.md

```diff
@@ -0,0 +1,313 @@
+---
+
+
+title: "Harbor Down/Harbor Unhealthy"
+
+  
+---
+
+This alert fires because a component (eg core, registry, portal etc) of Harbor is either down or unhealthy.
+
+When a component of Harbor is down it means that the pods are crashing off or failing to create.
+
+When a component of Harbor is unhealthy it means it is failing healthchecks. This could mean that the component either is not running or is is not ready.
+
+The first step is always to check the appropriate logs.
+
+```
+
+kubectl -n harbor-cluster logs harbor-$COMPONENT
+
+```
+
+Based on what the logs say try to figure out what is causing the error. Usually it will be the result of an improper configuration. Most configuration for the cluster is created in harbors [fullstack.yaml](https://github.com/goharbor/harbor-operator/blob/master/manifests/samples/full_stack.yaml). Below are some things to look out for if you cannot find the cause.
```

Comment:
```suggestion
Based on what the logs say try to figure out what is causing the error. Usually it will be the result of an improper configuration. Most configuration for the cluster is created in harbors [full_stack.yaml](https://github.com/goharbor/harbor-operator/blob/master/manifests/samples/full_stack.yaml). Below are some things to look out for if you cannot find the cause.
```